app/worker.js

```diff
@@ -27,14 +28,20 @@ class Worker {
     );
   }
 
-  fetchUpstream(request) {
+  fetchUpstream(urlOrRequest, init) {
+    let request = urlOrRequest instanceof Request
+      ? urlOrRequest
+      : new Request(urlOrRequest, init);
+
     const url = new URL(request.url);
-    const originalHost = url.host;
+    if (url.host === `localhost:${this.port}` || url.host === `127.0.0.1:${this.port}`) {
```

Comment:
This hardcoding of urls is what I was hoping to avoid. Hmmmm. Perhaps if we accept a regex of hosts to forward? This would probably best emulate cloudflare. 

For example, we have a multi tenant system, which we test with *.lvh.me.